srcpkgs/ncdc/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'ncdc'
+pkgname=ncdc
+version=1.20
+revision=1
+build_style=configure
+configure_args="--prefix=/usr --with-geoip"
+hostmakedepends="pkg-config glib-devel ncurses-devel sqlite-devel gnutls-devel geoip-devel"
+depends="bzip2"
```

Comment:
why is that needed?

Reply:
I don't rly know. I can see this in https://g.blicky.net/ncdc.git/tree/configure.ac
In this time I've add zlib-devel and bzip2-devel to makedepends

Reply:
Then that should be enough. `depends` would only be needed if the resulting propragm executed the bzip2 binary at runtime. It probably just links the library and uses that.